register/models.py

```diff
@@ -105,8 +109,21 @@ def invite(self, request):
     def send_reminder(self, request):
         if not request.user.has_perm('register.invite_application'):
             raise ValidationError('User doesn\'t have permission to invite thus can\'t send reminds neither')
+        if self.status != APP_INVITED:
+            raise ValidationError('Reminder can\'t be sent to none pending applications')
```

Comment:
Also, I think this message is misleading, as we have a pending status, and the condition is that the application is accepted (yes, pending to give an answer, but not in pending status)